packages/client/wallets/aa/src/blockchain/wallets/EVMAAWallet.ts

```diff
@@ -152,7 +142,7 @@ export class EVMAAWallet extends LoggerWrapper {
                             abi: erc20,
                             functionName: "transfer",
                             args: [toAddress, (config as ERC20TransferType).amount],
-                            ...this.getLegacyTransactionFeesParamsIfApply(),
+                            ...(usesGelatoBundler(this.chain) && gelatoBundlerProperties),
```

Comment:
This repetition is cleaned up [here](https://github.com/Crossmint/crossmint-sdk/pull/502)